test/performance/perf_test.lua

```diff
@@ -0,0 +1,467 @@
+            g.router:call("crud.enable_stats", {{ driver = 'metrics', quantiles = true }})
+        end,
+        column_name = column_name.metrics_quantile_stats,
+    },
+}
+
+local integration_params = {
+    timeout = 2,
+    fiber_count = 5,
+    connection_count = 2,
+}
+
+local insert_perf = {
+    timeout = 30,
+    fiber_count = 600,
+    connection_count = 10,
+}
+
+-- Higher load may lead to net_msg_max limit break.
+local select_perf = {
+    timeout = 30,
+    fiber_count = 200,
+    connection_count = 10,
+}
+
+local cases = {
+    crud_insert = {
+        call = 'crud.insert',
+        params = insert_params,
+        matrix = stats_cases,
+        integration_params = integration_params,
+        perf_params = insert_perf,
+        row_name = row_name.insert,
+    },
+
+    crud_insert_without_stats_wrapper = {
+        prepare = function(g)
+            g.router:eval([[
+                rawset(_G, '_plain_insert', require('crud.insert').tuple)
+            ]])
+        end,
+        call = '_plain_insert',
+        params = insert_params,
+        matrix = { [''] = { column_name = column_name.without_stats_wrapper } },
+        integration_params = integration_params,
+        perf_params = insert_perf,
+        row_name = row_name.insert,
+    },
+
+    crud_select_pk_eq = {
+        prepare = select_prepare,
+        call = 'crud.select',
+        params = select_params_pk_eq,
+        matrix = stats_cases,
+        integration_params = integration_params,
+        perf_params = select_perf,
+        row_name = row_name.select_pk,
+    },
+
+    crud_select_without_stats_wrapper_pk_eq = {
+        prepare = function(g)
+            g.router:eval("_plain_select = require('crud.select').call")
+            select_prepare(g)
+        end,
+        call = '_plain_select',
+        params = select_params_pk_eq,
+        matrix = { [''] = { column_name = column_name.without_stats_wrapper } },
+        integration_params = integration_params,
+        perf_params = select_perf,
+        row_name = row_name.select_pk,
+    },
+
+    crud_select_pk_gt = {
+        prepare = select_prepare,
+        call = 'crud.select',
+        params = select_params_pk_gt,
+        matrix = stats_cases,
+        integration_params = integration_params,
+        perf_params = select_perf,
+        row_name = row_name.select_gt_pk,
+    },
+
+    crud_select_without_stats_wrapper_pk_gt = {
+        prepare = function(g)
+            g.router:eval([[
+                rawset(_G, '_plain_select', require('crud.select').call)
+            ]])
+            select_prepare(g)
+        end,
+        call = '_plain_select',
+        params = select_params_pk_gt,
+        matrix = { [''] = { column_name = column_name.without_stats_wrapper } },
+        integration_params = integration_params,
+        perf_params = select_perf,
+        row_name = row_name.select_gt_pk,
+    },
+}
+
+local function fiber_generator(conn, call, params, report, timeout)
```

Comment:
Nit: The name confuses me a bit first time: the function does not generate a fiber. We sometimes name a fiber entry point as `*_f`. Personally I found this convention convenient.

Reply:
(Will not re-verify, just resolve.)